src/i18n/lang-en.ts

```diff
@@ -1,43 +1,43 @@
-import { Translation } from '../models';
+import { Translation } from "../models";
 
 export const translation: Translation = {
-  activate: 'Activate',
-  activated: 'Material Icon Theme is active.',
-  iconPacks: {
-    selectPack: 'Select an icon pack',
-    description: "Select the '%0' icon pack",
-    disabled: 'Disable icon packs',
-  },
-  folders: {
-    toggleIcons: 'Pick a folder theme',
-    color: 'Choose a folder color',
-    hexCode: 'Insert a HEX color code',
-    wrongHexCode: 'Invalid HEX color code!',
-    disabled: 'No folder icons',
-    theme: {
-      description: "Select the '%0' folder theme",
-    },
-  },
-  opacity: {
-    inputPlaceholder: 'Opacity value (between 0 and 1)',
-    wrongValue: 'The value must be between 0 and 1!',
-  },
-  toggleSwitch: {
-    on: 'ON',
-    off: 'OFF',
-  },
-  explorerArrows: {
-    toggle: 'Toggle folder arrows',
-    enable: 'Show folder arrows',
-    disable: 'Hide folder arrows',
-  },
-  grayscale: {
-    toggle: 'Toggle grayscale icons',
-    enable: 'Enable grayscale icons',
-    disable: 'Disable grayscale icons',
-  },
-  saturation: {
-    inputPlaceholder: 'Saturation value (between 0 and 1)',
-    wrongValue: 'The value must be between 0 and 1!',
-  },
+	activate: "Activate",
```

Comment:
Please format this part with [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode), to that we have a consistent formatting.

Reply:
Done.